notebooks/MAST/HSC/CASJOBS/HCV_API/HCV_API_demo.ipynb

```diff
@@ -0,0 +1,1039 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load_ext pycodestyle_magic\n",
+    "%flake8_on --ignore E261,E501,W291,W293"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Hubble Catalog of Variables Notebook (API version)\n",
+    "### September 2019, Rick White & Steve Lubow\n",
+    "\n",
+    "This notebook shows how to access the [Hubble Catalogs of Variables (HCV)](http://archive.stsci.edu/hlsp/hcv/).  The HCV is a large catalog of faint variable objects extracted from version 3 of the [Hubble Source Catalog](https://archive.stsci.edu/hst/hsc).  The HCV project at the National Observatory of Athens was funded by the European Space Agency (PI: Alceste Bonanos).  The data products for the HCV are available both at the [ESA Hubble Archive](http://archives.esac.esa.int/ehst) at [ESAC](https://www.cosmos.esa.int/web/esdc) through the [HCV Explorer](http://archives.esac.esa.int/hcv-explorer) interface and at STScI.\n",
+    "\n",
+    "This notebook uses the [MAST HSC catalog interface](https://catalogs.mast.stsci.edu/hsc), which supports queries to the current and previous versions of the [Hubble Source Catalog](https://archive.stsci.edu/hst/hsc). It allows searches of several different tables including the HCV summary and detailed tables. It also has an associated [API](https://catalogs.mast.stsci.edu/docs/hsc.html) that is used for data access in this notebook.\n",
+    "\n",
+    "For similar examples using the [MAST CasJobs](https://mastweb.stsci.edu/hcasjobs) interface,\n",
+    "a SQL database query interface that is more complex to use but more powerful than the API, see the [HCV_CasJobs_demo notebook](HCV_casjobs_demo.html).\n",
+    "\n",
+    "This notebook is available for [download](HCV_API_demo.ipynb)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Instructions: \n",
+    "* Complete the initialization steps [described below](#initialization).\n",
+    "* Run the notebook to completion. \n",
+    "* Modify and rerun any sections of the Table of Contents below.\n",
```

Comment:
This is a bit problematic from a pedagogical perspective.  "Run it all the way through and then think about it" is not nearly as good as "run it, trying to understand what each step does".  That said, I think it could also just be rephrased as "be sure to run the initialization steps before running the rest of the notebook" or similar.